modules/articles.py

```diff
@@ -0,0 +1,60 @@
+import nltk
+import newspaper
```

Comment:
Very good idea! I'll do that.

Although since our article object was named 'article', I think simply importing `from newspaper import Article` would be better.

Reply:
how do we pass the url of the article now ?